### PR TITLE
Added create, delete, and find privacy group endpoints

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added create, delete, and find privacy group endpoints

### CODE REVIEW

File: docs/Reference/Pantheon-API-Methods.md [hunk 1]
```diff
@@ -3769,6 +3769,113 @@ None
...
+    {
+      "jsonrpc": "2.0",
+      "id": 53,
+      "result": "ewuTVoc5nlvWMwTFdRRK/wvV0dcyQo/Pauvx5bNEbTk="
+    }
+    ```
+
+### eea_findPrivacyGroup
+
+Returns a list of privacy groups that contain exactly the listed members. For example, if the listed members
```
Comment (bgravenorst) [inline docs/Reference/Pantheon-API-Methods.md]: Returns a list of privacy groups that contain only...

File: docs/Reference/Pantheon-API-Methods.md [hunk 2]
```diff
@@ -3769,6 +3769,113 @@ None
...
+Returns a list of privacy groups that contain exactly the listed members. For example, if the listed members 
+are A and B, a privacy group containing A, B, and C is not returned. 
+
+**Parameters** 
+
+`array of data` - Members specified by Orion public keys 
+
+**Returns** 
+
+Privacy groups that contain exactly the specified members.
```
Comment (bgravenorst) [inline docs/Reference/Pantheon-API-Methods.md]: Privacy groups that contain only...

File: docs/Reference/Pantheon-API-Methods.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Reference/Pantheon-API-Methods.md]: Fixed

File: docs/Reference/Pantheon-API-Methods.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Reference/Pantheon-API-Methods.md]: Fixed

**Pull request merged**